docker/test-server/Dockerfile

```diff
@@ -6,12 +6,12 @@ ENV PIP_DISABLE_PIP_VERSION_CHECK=1
 ENV PIP_NO_CACHE_DIR=1
 
 # install packages and pip requirements first, in a single step,
-COPY /requirements.txt /requirements.txt
+COPY /requirements.in/base.txt /requirements.txt
```

Comment:
I don't understand this change. Can you explain?

Reply:
This is base requirement file (located in this directory: **/requirements.in/**), I just use that one, because /requirements.txt (in root of repo) is autogenerated file that's why I avoid that.